tests/test_search.py

```diff
@@ -102,7 +102,7 @@ def test_beam_search():
     results, mask, costs = search.search(
         {inputs: input_vals}, 0, 3 * length, as_arrays=True)
     # Just check sum
-    assert results.sum() == 3160
+    assert results.sum() == 1116
```

Comment:
Why the sum is different? It should be the same if the bias is eliminated by softmax.